sklearn/linear_model/ridge.py

```diff
@@ -351,14 +395,15 @@ def ridge_regression(X, y, alpha, sample_weight=None, solver='auto',
         # When y was passed as a 1d-array, we flatten the coefficients.
         coef = coef.ravel()
 
-    return coef
```

Comment:
The `ridge_regression` function is part of the public API so we cannot change the values returned by default. Please add a new kwarg named `return_n_iter=False` to make it explicit to get this behaviour. Also please update the docstring of this function to document the new return value and the new matching parameter.